CHANGES.rst

```diff
@@ -51,6 +51,11 @@ https://github.com/zopefoundation/Zope/blob/4.x/CHANGES.rst
 
 - Improve documentation for Zope's error logging services.
 
+- Add ``Paste`` as install dependency, otherwise it won't start when installed
+  with `pip` and the `constraints.txt` file.
+  (`#734 <https://github.com/zopefoundation/Zope/issues/734>`_)
```

Comment:
This is misleading. It only won't start if you used the built-in `mkwsgiinstance` to create the WSGI server `.ini` file.

Reply:
Changed the wording